metadata-ingestion/examples/recipes/dbt_to_datahub.yml

```diff
@@ -2,11 +2,11 @@
 source:
   type: "dbt"
   config:
-    manifest_path: "./your/dbt/manifest.json"
-    catalog_path: "./your/dbt/catalog.json"
-    target_platform: "[target data platform]"
-    load_schemas: True # or false
+    manifest_path: "./tests/integration/dbt/dbt_manifest.json"
+    catalog_path: "./tests/integration/dbt/dbt_catalog.json"
+    target_platform: "dbt"
+    load_schemas: True # or False
```

Comment:
should the example recipe be modified?

Reply:
Not totally necessary, but just made testing things out a bit easier. I've simplified the paths so people won't get too confused though.